README.md

```diff
@@ -13,10 +13,28 @@ Chris S Magnano, Anthony Gitter.
 
 ## Dependencies
 
-Pathway parameter advising was written and tested using Python 3.6 and requires the packages `networkx` and `numpy`.
-The `runPPA.sh` and `setupPGD.sh` scripts require git.
+Pathway parameter advising was written and tested using Python 3.6 and requires the packages `networkx`, `numpy`, and `requests`.
+
+Graphlet decomposition is performed using the PGD library.
+The PGD library can be installed from its [github repository](https://github.com/nkahmed/pgd) and complied using `make`.
+The script `setupPGD.sh` requires git, and has been provided to aid in installing the PGD library.
+However, it is not guaranteed to work and has only been tested on Ubuntu 18.04.
+
 These scripts have only been tested in a Linux environment initially.
 
+## Installation
+To download pathway parameter advising using pip, run the command:
+> 'pip download --no-deps --no-binary :all: pathwayParameterAdvising`
+
+And untar the resulting archive file. 
```

Comment:
Should we give the command to help novices?  `tar -xf pathwayParameterAdvising-*.tar.gz` or something similar?

Reply:
I added this to the README.